rollup.config.mjs

```diff
@@ -9,8 +10,9 @@ export default {
     preserveModules: true,
     preserveModulesRoot: 'src',
   },
-  external: ['react-children-by-type'],
+  external: [/node_modules/],
   plugins: [
+    nodeResolve(),
```

Comment:
What problem were you seeing? Node modules were included in the rollup build?

Reply:
Okay. Wasn't clear before, but I think I got it:

- By default rollup only includes relative modules in the bundle
- The node-resolve plugin lets us include node_modules in the bundle

With this new config, is React and/or Headless being included in the bundle? If so, should those be explicitly marked as external (instead of included in the bundle)?

Reply:
nvm, Jin and I looked into this more and all's good with it.

The plugin just tells rollup how to resolve paths to node_modules.

Reply:
Added a comment explaining what the external option does